.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,61 @@
+name: Bug Report
+description: Report an issue with existing supported devices.
+labels: ["bug"]
+body:
+  - type: input
+    id: info-device
+    attributes:
+      label: Affected Device
+      description: What is the name of the device you are using that is not functioning as expected? If more than one devices are experiencing issues list them here separated by a comma.
+      placeholder: e.g. Corsair Hydro H100i Pro XT
+    validations:
+      required: true
+  - type: input
+    id: info-distro
+    attributes:
+      label: Linux Distribution
+      description: Which Linux distribution are you using?
+      placeholder: ex. Arch
+    validations:
+      required: true
+  - type: input
+    id: install-method
+    attributes:
+      label: Installation Method
+      description: How did you install Liquidctl?
+      placeholder: e.g. Official Arch Linux repository 
+    validations:
+      required: true
+  - type: input
+    id: info-version
+    attributes:
+      label: Liquidctl Version
+      description: Output of `liquidctl --version`
+      placeholder: e.g. liquidctl v1.9.0 (Linux-5.16.16-arch1-1-x86_64-with-glibc2.35) 
+    validations:
+      required: true
```

Comment:
Not a major thing, but can you make both descriptions account for the scenario where the bug is in the installation procedure?

(But keeping them short & simple for the far more common case of a bug after the installation was successful).

Reply:
I moved this code around. I made a new comment highlighting the changes.